doit.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+set -e
+
+for i in {1..100}; do
+	echo "Attempt start: $i"
+	ginkgo -p --label-filter='simple' .
+	echo "Attempt end: $i"
+done
```

Comment:
Did you want this in there? It is handy.

Reply:
whoops. No, that was unintentional.